### ISSUE TITLE
Combat missions fail if you arrive late for payment. Really?

### ISSUE THREAD

**Issue by tropicalpioneer** [pioneerspacesim/pioneer#5180]
### Observed behaviour
Not a bug, but does it really make sense that after winning the fight, you fail the mission for a late payment?

### Steps to reproduce

__My pioneer version (and OS):__

**Comment by impaktor** [comment]
@robothauler thoughts?

**Comment by robothauler** [comment]
In my opinion, the current behavior makes sense. If the Police plan a chain of operations, they may want to analyze the data before going to the next step. Especially when they hire a mercenary. If they get the data too late, the entire plan might get messed up.

**Comment by tropicalpioneer** [comment]
This means that the "primary mission objective ok" message in CommLog during battle is not sent by the hiring institution, otherwise they'd know the job is done.

Does that mean that I physically need to be proving the killings? For example the hiring institution needs to read the data from a kind of black box or other incorruptible device on my ship.

Otherwise, if I do not need to be physically there, how about a 3220AD interstellar "phone call" :) to keep the chain of operations updated, then wire the money _whenever_ I am in range?

**Comment by tropicalpioneer** [comment]
oh, and when the payment appointment is in open space, who am I talking with?
Is there a ship waiting somewhere after hyperjump, to get in range with, chit chat the adventure around a floating spherical beer (or a black box full of data) and getting paid?

**Comment by robothauler** [comment]
> This means that the "primary mission objective ok" message in CommLog during battle is not sent by the hiring institution, otherwise they'd know the job is done.

It's a message from the ship's mission computer to the pilot. And yes, the CommsLog should indicate better who is saying what.

> Does that mean that I physically need to be proving the killings? For example the hiring institution needs to read the data from a kind of black box or other incorruptible device on my ship.

In my opinion, yes.

> Otherwise, if I do not need to be physically there, how about a 3220AD interstellar "phone call" :) to keep the chain of operations updated, then wire the money _whenever_ I am in range?

The speed of light is too slow. The hyperdrive is faster! :-)

**Comment by robothauler** [comment]
> oh, and when the payment appointment is in open space, who am I talking with and passing the data to?
> Is there a ship waiting somewhere after hyperjump, to get in range with, chit chat the adventure around a floating spherical beer (or a black box full of data) and getting paid?

Yes, there is a ship! I noticed that sometimes the time acceleration didn't stop after entering the target system. So the ship jumps out and you see only the hyperdrive departure cloud in your vicinity.

**Comment by tropicalpioneer** [comment]
I am new to hyper-drive thinking and I get the speed relation now. Funtastic! :)

Maybe there is a way to make the post hyper-jump meeting more tangible? 
Would it be too much programming/navigation if we have to get to closer to that waiting ship? (thinking about those 50m needed in physical transfers: staff, fuel, ...)

Will you pull a request to make explicit the message senders in CommsLog?

**Comment by tropicalpioneer** [comment]
I added a [feature request](https://pioneerspacesim.net/forum/viewtopic.php?f=3&t=665) about the CommsLog, about making it easier to read the latest entries.
